respa_admin/static_src/js/periods.js

```diff
@@ -39,6 +52,25 @@ function initialSortPeriodDays() {
   }
 }
 
+function enableTemplateCopyButton() {
```

Comment:
The template select dropdown allows multiple options, so loop through selected templates and copy into the form.

The options should be unchecked and the copy button hidden once the templates are copied.